tests/requirements-relaxed.in

```diff
@@ -0,0 +1,13 @@
+# This requirements file containes no pins on antsibull-docs and other
+# essential tools
+# It's used for testing purposes and not for the production docs.ansible.com
+# builds.
+
+jinja2 >= 3.0.0 # used by hacking/build_library/build_ansible/command_plugins/generate_man.py and dump_keywords.py
+pyyaml >= 5.1 # used by ansible-core
+resolvelib >= 0.5.3, < 1.1.0 # used by ansible-core
```

Comment:
Same here. It's a transitive dependency, it should be in a constraints file.

Reply:
I'm not sure why we have some constraints in one file and others in another. 

Anyways, if we need to grab stuff from ansible-core, what do you think about adding `-r ../requirements.txt` to pull from the ansible-core requirements.txt created by `docs/bin/clone-core.py`?

Reply:
> I'm not sure why we have some constraints in one file and others in another.

There are inputs to pip-compile and outputs. Requirements declare direct dependencies (as in things that we use known features from). Then, there's known broken versions of (possibly transitive) dependencies. We want to exclude them from the dependency resolution, but not confusingly mark them as direct dependencies. All of these are inputs.
What pip-compile output is the true lockfile. But I still want to separate semantically differing things by not dumping them into the same place.

> Anyways, if we need to grab stuff from ansible-core, what do you think about adding `-r ../requirements.txt` to pull from the ansible-core requirements.txt ~created~ copied by `docs/bin/clone-core.py`?

I was considering this, but this will not work with Dependabot since it doesn't know about other repos and would just break. Having all the input files here allows managing the lockfiles without depending on unexpected external states.